docs/src/App.js

```diff
@@ -0,0 +1,97 @@
+import './App.css';
+import {useState} from "react";
+import DeleteCompletedAlert from "./DeleteCompletedAlert";
+import ListItems from "./ListItems";
+import {generateUniqueID} from "web-vitals/dist/modules/lib/generateUniqueID";
+
+function App(props) {
+    const [items, setItems] = useState(props.initialData);
+    const [completedItems, setCompletedItems] = useState([]);
+    const [showDeleteAlert, setShowDeleteAlert] = useState(false);
+    // const deletedItems = [];
+    const [totalItems, setTotalItems] = useState(props.initialData);
```

Comment:
If you want to keep two lists: one with all items, and one only with completed, then have a single state variable that stores all of them, and then compute the ones with completed (by doing a filter) in your component.

Don't have two state variable that you have to try to synchronize.

Reply:
Since a filter within a set method, sets the filtered list as the new "items" object, how would we retrieve the completed items lost in the filter without using a separate list that doesn't get filtered?